source/Halibut.Tests/MaximumAuthorizedTcpConnectionsFixture.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Halibut.Diagnostics;
+using Halibut.Tests.Support;
+using Halibut.Tests.Support.TestAttributes;
+using Halibut.Tests.Support.TestCases;
+using Halibut.Tests.TestServices.Async;
+using Halibut.TestUtils.Contracts;
+using NUnit.Framework;
+
+namespace Halibut.Tests
+{
+    public class MaximumAuthorizedTcpConnectionsFixture : BaseTest
+    {
+        [Test]
+        [LatestClientAndLatestServiceTestCases(testListening: false, testPolling: true, testWebSocket: false, testNetworkConditions: false)]
+        public async Task WhenLimitIsExceededSubsequentConnectionsAreRejected(ClientAndServiceTestCase clientAndServiceTestCase)
```

Comment:
Does this actually test anything?

Reply:
Maybe the test is very much in draft

Reply:
correct. I was validating via logs that it was at least working. I'll work on getting a valid test working in a minute

Reply:
Added checking the logs in [ab05e55](https://github.com/OctopusDeploy/Halibut/pull/599/commits/ab05e55c578e1bb5886d27b38ac991a9f46602ac)

I will sync with @LukeButters to discuss a more comprehensive way to do this as log checking feels icky